spec/models/user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
```

Comment:
- [ ] Custom methods in the models should have corresponding unit tests. Please test the `recent_post` method in your User model